transaction-service/src/main/java/pl/adamboguszewski/transactionservice/application/Transaction.java

```diff
@@ -0,0 +1,58 @@
+package pl.adamboguszewski.transactionservice.application;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.util.List;
+
+public class Transaction {
+
+    private final List<Product> products;
+
+    private final double totalPrice;
+
+    private final String transactionId;
```

Comment:
Product id: id
Transaction id: transactionId
🤔
Same thing as in product class, Long id as database id and UUID transactionId as internal id